src/elm/Api.elm

```diff
@@ -447,26 +432,6 @@ updateCurrentUser model body =
         |> withAuth model.session
 
 
-{-| getRepositories : fetches enabled repositories by user token
--}
-getRepositories : PartialModel a -> Maybe Pagination.Page -> Maybe Pagination.PerPage -> Request Repositories
-getRepositories model maybePage maybePerPage =
-    get model.velaAPI (Endpoint.Repositories maybePage maybePerPage) decodeRepositories
-        |> withAuth model.session
-
-
-{-| getAllRepositories : used in conjuction with 'tryAll', it retrieves all pages of the resource
-
-    Note: the singular version of the type/decoder is needed in this case as it turns it into a list
-
--}
-getAllRepositories : PartialModel a -> Request Repository
-getAllRepositories model =
-    -- we using the max perPage setting of 100 to reduce the number of calls
-    get model.velaAPI (Endpoint.Repositories (Just 1) (Just 100)) decodeRepository
-        |> withAuth model.session
```

Comment:
I know this is in Draft but we will be using `getAllRepositories` in the Overview refactor coming soon 👍

Reply:
Readded